ONSdigital/eq-questionnaire-runner
# Migrate to wdio assertions library

### What is the context of this PR?
The purpose of this PR is to remove the usages of chai and use wdio's builtin assertion library to make assertions work as intended

### How to review
- Run functional tests to see if passes
- Check the changes to ensure webdriver syntax is used and is correct
- Potentially converting a single file with the tool but this may be difficult as manual intervention is required and I added my own improvements which isn't done by the tool so the results will differ


### Checklist
